src/App.svelte

```diff
@@ -1,5 +1,6 @@
 <script>
-  import Card from "./Card.svelte";
+  import { dataset_dev } from "svelte/internal";
+import Card from "./Card.svelte";
```

Comment:
could you fix the indentation on this line?

Reply:
Will do  kinda late today 😅

Reply:
no worries, thank you 👍

Reply:
The moderators part has been changed 

I think the best thing would be to create a separate component for it.. will be doing it after responsiveness is fixed .. will be more organised

Reply:
yes a separate component is better. When you refactor to a separate component this logic becomes easier to reason about 👍 : 
![imagem](https://user-images.githubusercontent.com/18630253/94376591-57901980-0113-11eb-815e-aa70c0927e06.png)

What you can do to simplify those conditions is to make the component receive a list of labels, where each label has its name and color. Then when we create each card we specify the labels we want 😃. For example, Stephen would have the "Server booster", and Mikey would have "python".
Do you agree with this solution?

Reply:
I tried to keep it as simple as possible to encourage participation , since most ppl don't know svelte 
Stephen is in a different div cuz his button styling is different :)